datalake/src/main/java/com/sequenceiq/datalake/flow/dr/event/DatalakeDatabaseRestoreCouldNotStartEvent.java

```diff
@@ -0,0 +1,33 @@
+package com.sequenceiq.datalake.flow.dr.event;
+
+import com.sequenceiq.datalake.flow.SdxEvent;
+
+public class DatalakeDatabaseRestoreCouldNotStartEvent extends SdxEvent {
+    private final Exception exception;
+
+    public DatalakeDatabaseRestoreCouldNotStartEvent(Long sdxId, String userId, Exception exception) {
+        super("DatalakeDatabaseRestoreCouldNotStartEvent", sdxId, userId);
+        this.exception = exception;
+    }
+
+    public static DatalakeDatabaseRestoreCouldNotStartEvent from(SdxEvent event, Exception exception) {
+        return new DatalakeDatabaseRestoreCouldNotStartEvent(event.getResourceId(), event.getUserId(), exception);
+    }
+
+    @Override
+    public String selector() {
+        return "DatalakeDatabaseRestoreCouldNotStartEvent";
+    }
+
+    public Exception getException() {
+        return exception;
+    }
+
+    @Override
+    public String toString() {
+        final StringBuffer sb = new StringBuffer("DatalakeDatabaseRestoreCouldNotStartEvent{");
```

Comment:
you should use either plain String with concatenation or `StringBuilder`.
also final is unnecessary

Reply:
I will remove the final.
StringBuilder is not thread-safe by StringBuffer is. That is why I used StringBuffer.
Is there a specific reason to use StringBuilder?

Reply:
why would you need thread-safe solution here? it has a performance drawback
also simple string concatenation would do, I usually use that for toString

Reply:
will do